app/assets/javascripts/admin/addon/components/admin-user-field-item.hbs

```diff
@@ -133,6 +133,11 @@
             </group.Field>
           </form.CheckboxGroup>
 
+          <PluginOutlet
+            @name="after-admin-user-fields"
+            @outletArgs={{hash userField=@userField form=form}}
```

Comment:
This used to have a single argument `buffered`, but with the new implementation this needs to change. Update to the plugin is incoming.